pkgs/by-name/li/libskk/package.nix

```
@@ -14,6 +14,15 @@ stdenv.mkDerivation rec {
     sha256 = "0y279pcgs3jrsi9vzx086xhz9jbz23dqqijp4agygc9ackp9sxy5";
   };
 
+  patches = [
+    # fix parse error in default.json
+    # https://github.com/ueno/libskk/pull/90
+    (fetchpatch {
+      url = "https://github.com/ueno/libskk/commit/2382ebedc8dca88e745d223ad7badb8b73bbb0de.diff";
+      sha256 = "sha256-e1bKVteNjqmr40XI82Qar63LXPWYIfnUVlo5zQSkPNw=";
```

```suggestion
      hash = "sha256-e1bKVteNjqmr40XI82Qar63LXPWYIfnUVlo5zQSkPNw=";
```
nitpick
We should use `hash` instead of `sha256`, but we can fix it treewide and leave it as is for now.
(It is a waste of time to spend another 10+ hours ofborg eval.🙃)